pkg/usecase/builder/build_buildkit.go

```diff
@@ -139,6 +139,20 @@ func (s *builderService) buildRuntimeCmd(
 	ch chan *buildkit.SolveStatus,
 	bc *domain.BuildConfigRuntimeCmd,
 ) error {
+	// if .dockerignore exists, rename it
```

Comment:
今回は少し複雑な背景からの変更になるので、次見たときに分かるように「why」をコメントしておきたいです。
自分だったら↓みたいに少し長めにしっかり書くかもなので、参考にできたらお願いします
```suggestion
	// If .dockerignore exists, rename to prevent it from being picked up by buildkitd,
	// as this is not the behavior we want in 'Command' build which is supposed to execute commands against raw repository files.
```